polaris-service/src/test/java/org/apache/polaris/service/admin/PolarisServiceImplIntegrationTest.java

```diff
@@ -99,9 +100,11 @@ public class PolarisServiceImplIntegrationTest {
   private static String realm;
 
   @BeforeAll
-  public static void setup(PolarisConnectionExtension.PolarisToken adminToken) throws IOException {
+  public static void setup(
+      PolarisConnectionExtension.PolarisToken adminToken, @PolarisRealm String realm)
+      throws IOException {
     userToken = adminToken.token();
-    realm = PolarisConnectionExtension.getTestRealm(PolarisServiceImplIntegrationTest.class);
+    PolarisServiceImplIntegrationTest.realm = realm;
```

Comment:
nit: I'd personally remove the field qualifier here as it was not present before.

Reply:
Updated to remove it in a couple of places.

Reply:
thx :+1: